Components/Modal/Modal.css

```diff
@@ -0,0 +1,48 @@
+.modal {
+  position: relative;
+}
+
+.modal-display {
+  padding: 2rem;
+  width: fit-content;
+  background-color: hsl(var(--clr-dark));
+  color: hsl(var(--clr-white));
+  cursor: pointer;
+}
+
+.modal-display:hover {
+  background-color: hsl(var(--clr-white));
+  color: hsl(var(--clr-dark));
+}
+.modal-fixed {
+  position: fixed;
+  inset: 0;
+  background-color: hsl(var(--clr-dark) / 0.7);
+  color: hsl(var(--clr-white));
+  padding: 4rem;
+  z-index: 99999;
+  backdrop-filter: blur(0.25rem);
```

Comment:
use px here?

Reply:
rems are fine i guess, depends from person to person